bindings/python/cconfigspace/base.py

```diff
@@ -7,9 +7,15 @@ class ccs_version(ct.Structure):
               ("minor",    ct.c_ushort),
               ("major",    ct.c_ushort)]
 
-  def __str__(self):
+  def __repr__(self):
     return "{}.{}.{}.{}".format(self.major, self.minor, self.patch, self.revision)
```

Comment:
Should be switched to the new format.
```suggestion
    return str(f"{self.major}.{self.minor}.{self.patch}-{self.revision}")
```
Also added the more classical "x.y.z-w" naming scheme.